lib/danger/request_sources/github/github.rb

```diff
@@ -40,6 +40,10 @@ def get_pr_from_branch(repo_name, branch_name, owner)
         end
       end
 
+      def validates_as_ci?
+        true
+      end
```

Comment:
In the base class, it is looking up the git remote origin to see if it contains `github.com`:
https://github.com/danger/danger/blob/61a09f84965edbb3a2150e5b1464fa6e39939fa7/lib/danger/request_sources/request_source.rb#L39-L42


Unfortunately, `RequestSources::Github#validates_as_ci?` is tricky now because
the `git remote -v` returns origin `gitlab.com` rather than `github.com`. The
reasoning is that even though the actual repo is on `Github.com`, GitlabCI maintains a
mirror on `Gitlab.com` and the CI checkout the code from there.

I could change this to also accept an origin that matches gitlab.com if the `ci_source` is Gitlab, however I am not sure I understand why there is a need to validate this at all, especially if we are already performing the `validates_as_api_source?`.

Reply:
I think this is probably reasonable